recipes-kernel/linux/linux-mainline_5.0.2.bb

```diff
@@ -19,10 +19,10 @@ KERNEL_EXTRA_ARGS += "LOADADDR=${UBOOT_ENTRYPOINT}"
 
 S = "${WORKDIR}/linux-${PV}"
 	
-SRC_URI[md5sum] = "86f56f586f35ac3a98e086dccfd2ff99"
-SRC_URI[sha256sum] = "6f0a7b3abd48eca3df5e29bfbcfc1c06dd2792f86d23cdb9ad37cf3e572df2e3"
+SRC_URI[md5sum] = "911065be61e90bc9afefd2c0ffa944c1"
+SRC_URI[sha256sum] = "43bfea3a6b24b4e5f63190409a199bee8cb93dbea01c52ad7f017078ebdf7c9b"
 
-SRC_URI = "https://www.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz \
+SRC_URI = "https://www.kernel.org/pub/linux/kernel/v5.x/linux-${PV}.tar.xz \
```

Comment:
Why updating to 5.0.2? Is there some specific reason for that? Thanks.

Reply:
The kernel version which was in this meta-layer at that time did not have a correct device-tree. And back-porting the devicetree introduced a lot of problems. However since the latest stable kernel right now is 5.0.19, this commit can be seen as void :)